contracts/libs/StructuredLinkedList.sol

```diff
@@ -0,0 +1,235 @@
+pragma solidity ^0.5.7;
+
+contract StructureInterface {
```

Comment:
Think you can import this from npm https://truffleframework.com/docs/truffle/getting-started/package-management-via-npm#within-your-contracts